sky/templates/azure-ray.yml.j2

```diff
@@ -87,17 +87,18 @@ setup_commands:
   - pip3 --version > /dev/null 2>&1 || (curl -sSL https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py && echo "PATH=$HOME/.local/bin:$PATH" >> ~/.bashrc);
     (type -a python | grep -q python3) || echo 'alias python=python3' >> ~/.bashrc;
   - which conda > /dev/null 2>&1 || (wget -nc https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh && bash Miniconda3-latest-Linux-x86_64.sh -b && eval "$(/home/azureuser/miniconda3/bin/conda shell.bash hook)" && conda init && conda config --set auto_activate_base true);
+  # patch the buggy ray files and enable `-o allow_other` option for `goofys`
```

Comment:
Maybe keep this section the same across clouds? AWS' template has an additional `# This also kills...` comment.